d2l-icon-button-shared-styles.html

```diff
@@ -0,0 +1,54 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../d2l-colors/d2l-colors.html">
+<dom-module id="d2l-icon-button-shared-styles">
+	<template>
+		<style include="d2l-colors">
+			:host {
+				--d2l-icon-button: {
+					background-color: transparent;
+					border: none;
+					cursor: pointer;
+					display: inline-block;
+					font-family: inherit;
+					font-size: inherit;
+					line-height: 0;
+					margin: 0;
+					padding: 0;
+					text-decoration: none;
+				};
+				--d2l-icon-button-hover: {
+					outline-style: none;
+				};
+				--d2l-icon-button-icon: {
+					height: 18px;
+					transition: color 0.3s ease;
+					width: 18px;
```

Comment:
What happens if we don't set the size here and just let the icon size itself based on which tier was used?